libopenage/unit/unit.h

```diff
@@ -175,6 +175,12 @@ class Unit : public log::LogSource {
 	void add_attribute(std::shared_ptr<AttributeContainer> attr);
 
 	/**
+	 * delete a given attribute this this unit
+	 * this is used to delete/change things like color, hitpoints and speed
```

Comment:
I noticed this at the time, but (most of) the rest of the doccomments in this file have a similar lack of punctuation. something to fix elsewhere perhaps?